index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
We have that great domain name - one that people can remember. Search engines also very much prefer when the domain name matches the product name. All good there. Also, we'll deal with it shortly with metadata in _config.yml, courtesy of an SEO gem, instead of actual text. Can put whatever your heart desires in there, without consequence.